external/bart/bart/sched/functions.py

```diff
@@ -139,6 +139,25 @@ def sanitize_asymmetry(series, window=None):
                 if window:
                     series.index.values[-1] = window[1]
 
+            # Remove repeated entries - which could happen if a task switch in
+            # then immediately switches out; ie: time stamp is exactly the same
+            n = 0
+            next = n + 1
```

Comment:
Could we rename `next` to another name (or `next_` if we lack inspiration) ? `next()` is a builtin function so it's better to avoid overriding these names.

Reply:
Sure good point